onboarding/src/pages/BlankPage.tsx

```diff
@@ -1,9 +1,5 @@
-import React from 'react'
+import React from 'react';
 
 export default function BlankPage() {
```

Comment:
O eslint não xingou por exportar default? 
Outro ponto é, por padrão, mantemos sempre a nomenclatura de const, com arrow function e react.fc, ai quem for usar importa a const e segue a vida.
Isso acontece pq as vezes precisamos importar alguma outra coisa da página (uma props, por exemplo, ou uma outra const) e como estamos exportando como default não conseguimos.

```suggestion
export const  BlankPage: React.FC =() => {
```

Reply:
Inclusive acho q essa blankPage nem tá sendo usada, né?